SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -0,0 +1,31 @@
+# Dynamic Typebindings
+To allow Unreal to replicate through the SpatialOS network stack and combine multiple dedicated server instances across one seamless game world, we needed to adapt Unreal networking functionality to work with the [SpatialOS Worker API (SpatialOS documentation)](https://docs.improbable.io/reference/latest/capi/introduction). We did this seamlessly using `Dynamic Typebindings`. `Dynamic Typebindings` operate at runtime so that your iteration speed is not affected despite your network code running on a completely different representation than Unreal’s.
+
+At the heart of `Dynamic Typebindings` are the dynamically generated [SpatialOS schema](https://docs.improbable.io/reference/latest/shared/schema/introduction), which are the schema representation of any Unreal object and its replicated data and RPCs. `Dynamic Typebindings` also include the binding code that is invoked when converting network relevant data between native Unreal and SpatialOS.
+
+# Schema
+The schema used in `Dynamic Typebindings` are generated via the Unreal Editor, for all classes tagged with the [SpatialType]({{urlRoot}}/content/spatial-type) specifier. For each Unreal object that has schema generated for it, there are a number of possible schema components generated, each serving a different function.
+
+For Unreal Actors and subobjects:
+
+* Replicated property schema component (eg. _MyActor_): Contains all the replicated properties (including inherited) present on the object, except those tagged with the `COND_OwnerOnly` or `COND_AutonomousOnly` replication condition.
```

Comment:
This was lost in an update earlier: If IIRC we should not use eg. in the docs use for example instead, @ElleEss to correct me if I'm wrong

Reply:
...schema component (for example,  `MyActor`): Contains all the replicated properties present on the object. This includes inherited properties but excludes any properties tagged with...

Reply:
"for example" is our standard to make us look all professional and use the same terms across our docs. (Like grown-ups.)